doc/content/microreactors/gcmr/GCMR_results.md

```diff
@@ -0,0 +1,30 @@
+## Results - Dynamic Multiphysics simulation of flow blockage
+
+Total flow blockage of one channel at the center of the GC-MR assembly was simulated by time-dependent Multiphysics coupling of Griffin, BISON and SAM. The simulated transient models a localized flow-blockage transient in only one channel. In such scenario, only the temperature near the blocked channel increases with a slight drop in reactor power. As shown in [Fig_4] and [Fig_5], the relaxation time is relatively small compared since all the other cooling channels are still functional after the transient occurs. It only takes ~400 seconds for the system to reach new equilibrium. The assembly power is predicted to drop by only ~10 kW, while the local maximum temperature increases by ~40 K. The system reaches a new equilibrium after ~500 sec. The 3-D power and temperature distributions after 500 sec are demonstrated in [Fig_6].
```

Comment:
```suggestion
Total flow blockage of one channel at the center of the GC-MR assembly was simulated by time-dependent Multiphysics coupling of Griffin, BISON and SAM. The simulated transient models a localized flow-blockage transient in only one channel. In such scenario, only the temperature near the blocked channel increases with a slight drop in reactor power. As shown in [Fig_4] and [Fig_5], the relaxation time is relatively small compared since all the other cooling channels are still functional after the transient occurs. It only takes ~400 seconds for the system to reach new equilibrium. The assembly power is predicted to drop by only ~10 kW, while the local maximum temperature increases by ~40 K. The system reaches a new equilibrium after ~500 s. The 3-D power and temperature distributions after 500 s are demonstrated in [Fig_6].
```